vignettes/using-the-dccvalidator-app-amp-ad.Rmd

```diff
@@ -38,52 +31,96 @@ Some portions of the app submit data to Synapse. This allows curators at Sage to
 troubleshoot issues if needed; no one outside the Sage curation team will be
 able to download the data.
 
-## Documentation upload
+# Instructions
+
+## Step 1: Contact the AMP-AD data coordinating administrator
+
+Please contact the AMP-AD data coordinating
+administrator(<AMPAD_SageAdmin@synapse.org>) to let us know that you plan to
+submit data. The AMP-AD Knowledge Portal organizes data into studies, which are
+sets of data and metadata from a specific human cohort, model systems, or other
+experimental units If you are contributing data for a new study, we will work
+with you to name the study and provide you with a location to which you will
+upload data. If you are contributing new data for an existing study, we will
+provide you with the location to which you will upload data. If you're not sure
+whether your data should be a new study or part of an existing one, we can help
+you figure that out.
+
+## Step 2: Provide documentation of the study
+
+Each study has accompanying [documentation in the
+portal](https://adknowledgeportal.synapse.org/#/Explore/Studies?Study=syn8391648).
+If you are submitting a new study, or data for a new assay within an existing
+study, you will need to provide documentation about the study and of the methods
+you used. If you are submitting new data for an existing assay in an existing
+study (such as RNAseq data on additional donors), you can skip this step.
 
-### Metadata requirements
+You can submit your study and assay documentation through the dccvalidator app
+on the Documentation page. There must be a study description, and a description
+of each of the assays that were performed. These can be in a single file, or you
+can upload multiple files to the assay description section. 
+![A screenshot of a website with instructions on how to provide documentation of a study, and a space to upload documentation files](../man/figures/amp-documentation.png).
 
-Each study should include metadata that would help a new researcher understand
-and reuse the data. In most cases, we will expect 4 files:
+If you are submitting documentation for multiple studies, it should be submitted
+one study at a time. To acknowledge your data contribution you can upload an
+acknowledgement statement, and/or publication that should be referenced in
+publications based on secondary data use, together with the study and assay
+documentation.
```

Comment:
The acknowledgement should be within one of the files, either study documentation or one of the assay files. I think the way this reads might be a little confusing. It sounds as if there should be a separate file for the acknowledgement, but we don't provide a separate upload input for that. It also sounds as if they should upload the actual publication they want referenced.

>To acknowledge your data contribution you can add an
acknowledgement statement, and/or a preferred publication citation that should be referenced in
publications based on secondary data use, within the study or assay
documentation.